perfdash/downloader.go

```diff
@@ -53,12 +48,12 @@ func serveHTTPObject(res http.ResponseWriter, req *http.Request, obj interface{}
 	if err != nil {
 		res.Header().Set("Content-type", "text/html")
 		res.WriteHeader(http.StatusInternalServerError)
-		res.Write([]byte(fmt.Sprintf("<h3>Internal Error</h3><p>%v", err)))
+		fmt.Fprintln(res, "<h3>Internal Error</h3><p>%v", err)
```

Comment:
Why do we need to change this?

Reply:
This is fail on - errcheck lint,

perfdash/downloader.go:51:12: Error return value of `res.Write` is not checked (errcheck)
		res.Write([]byte(fmt.Sprintf("<h3>Internal Error</h3><p>%v", err)))